Makefile

```diff
@@ -685,6 +685,8 @@ endif
 docs-spellcheck: /usr/local/bin/mdspell
 	# check docs for spelling mistakes
 	mdspell --ignore-numbers --ignore-acronyms --en-us --no-suggestions --report $(shell find docs -name '*.md' -not -name upgrading.md -not -name README.md -not -name fields.md -not -name upgrading.md -not -name swagger.md -not -name executor_swagger.md -not -path '*/cli/*')
+	# alphabetize spelling file -- ignore first 5 lines (comments), then sort the rest case-sensitive and remove duplicates
```

Comment:
I just removed most of the comments since they weren't useful and were copy+pasted from the tool's [README](https://github.com/lukeapage/node-markdown-spellcheck/blob/78ad8e3c21fe982a1e4d1fc1fbf421fb57c9739b/readme.md?plain=1#L106).

Left only 1 comment specifying that it's a dictionary, so I think a second comment saying to edit this line is no longer necessary as there's only 1 comment and it doesn't really change